lmdeploy/turbomind/turbomind.py

```diff
@@ -155,6 +323,76 @@ def create_instance(self, cuda_stream_id=0):
         """
         return TurboMindInstance(self, cuda_stream_id)
 
+    @classmethod
+    def from_pretrained(cls,
+                        pretrained_model_name_or_path: str,
+                        model_name: Optional[str] = None,
+                        model_format: Optional[str] = None,
+                        group_size: Optional[int] = None,
+                        tp: Optional[int] = None,
+                        **kwargs):
+        """LMDeploy's turbomind inference engine.
+
+        Args:
+            pretrained_model_name_or_path (str):
+                Can be either:
+                    - 1) A path to a *directory* containing model which is
+                      converted previously by `lmdeploy convert` command.
+                    - 2) A repo_id or path which is managed by lmdeploy,
```

Comment:
- 2) The *model_id* of a quantized model hosted inside a model repo on huggingface.co/lmdeploy
- 3) The *model_id* of a model hosted inside a model repo on huggingface.co, such as "InternLM/internlm-chat-7b", "Qwen/Qwen-7B-Chat ", "baichuan-inc/Baichuan2-7B-Chat" and so on.